glean-core/src/metrics/string.rs

```diff
@@ -51,18 +51,7 @@ impl StringMetric {
             return;
         }
 
-        let s = value.into();
-        let s = if s.len() > MAX_LENGTH_VALUE {
-            let msg = format!(
-                "Value length {} exceeds maximum of {}",
-                s.len(),
-                MAX_LENGTH_VALUE
-            );
-            record_error(glean, &self.meta, ErrorType::InvalidValue, msg);
-            s[0..MAX_LENGTH_VALUE].to_string()
-        } else {
-            s
-        };
+        let s = truncate_string_at_boundary_with_error(glean, &self.meta, value, MAX_LENGTH_VALUE);
```

Comment:
Would you kindly also update [experiments](https://github.com/mozilla/glean/blob/104d49e8d9fd215e02469c09f6c0a8ab4e63f5e6/glean-core/src/metrics/experiment.rs#L60)?

Reply:
Thanks.  Missed that one.